gensim/models/doc2vec.py

```diff
@@ -59,16 +60,29 @@
 
 try:
     from gensim.models.doc2vec_inner import train_document_dbow, train_document_dm, train_document_dm_concat
-    from gensim.models.word2vec_inner import FAST_VERSION  # blas-adaptation shared from word2vec
+    # blas-adaptation shared from word2vec
+    from gensim.models.word2vec_inner import FAST_VERSION
     logger.debug('Fast version of {0} is being used'.format(__name__))
 except ImportError:
     logger.warning('Slow version of {0} is being used'.format(__name__))
-    # failed... fall back to plain numpy (20-80x slower training than the above)
+    # failed... fall back to plain numpy (20-80x slower training than the
+    # above)
```

Comment:
Are we really strictly enforcing 72-79 char max line lengths? (That's one of the most-often waived odler-standards, given the prevalence of larger/wider/laptop-aspect-ratio screens.)